.github/workflows/release.yml

```diff
@@ -86,15 +70,12 @@ jobs:
         with:
           subject-path: "${{ env.NAME }}-${{ env.VERSION }}.vsix"
 
-      - name: 📌 Push tags
-        run: git push
-
       - name: 🚀 Release ${{ env.VERSION }}
         uses: softprops/action-gh-release@01570a1f39cb168c169c802c3bceb9e93fb10974 # v2.1.0
         with:
           files: ${{ env.NAME }}-${{ env.VERSION }}.vsix
           tag_name: v${{ env.VERSION }}
-          body: ${{ env.RELEASE_NOTES }}
+          body_path: CHANGELOG.md
```

Comment:
I don't think it's appropriate to shove a whole changelog into the release notes. We need to extract the release notes only for the new version.

Reply:
Good point. I decided to use changelogen also to create the GH release, as mentioned in [their docs](https://github.com/unjs/changelogen?tab=readme-ov-file#changelogen-gh-release). I've updated the PR with 450135b